CreeDictionary/CreeDictionary/settings.py

```diff
@@ -250,14 +282,13 @@
 
 ############################## staticfiles app ###############################
 
-if RUNNING_ON_SAPIR:  # pragma: no cover
-    # on sapir /cree-dictionary/ is used to identify the service of the app
-    # XXX: this is kind of a hack :/
-    STATIC_URL = "/cree-dictionary/static/"
-else:
-    STATIC_URL = "/static/"
+STATIC_URL = env(
+    "STATIC_URL",
+    # XXX: hack to use the correct URL on Sapir if this setting is not explicitly set
+    default="/cree-dictionary/static/" if RUNNING_ON_SAPIR else "/static/",
```

Comment:
This probably isn’t too hard to fix, let’s give it a shot the next time we pair.